### ISSUE TITLE
Cannot reproduce the results on DomainNet when K = 1

### ISSUE THREAD

**Issue by Openning07** [VisionLearningGroup/SSDA_MME#5]
Thank for sharing your project~
Based on the data split and settings of hyper-parameters in the paper, I can obtain similar classification performance on DomainNet when K = 3, but not when K = 1 (e.g., in P to R, I get **73.65** accuracy, instead of **76.1** claimed in the paper; in R to S, I get **59.655** accuracy, rather than **61.0** claimed in the paper).

Do I miss some important details ?

**Comment by ksaito-ut** [comment]
Hi, are you using pytorch 0.4.0 to reproduce the results?

**Comment by Openning07** [comment]
> Hi, are you using pytorch 0.4.0 to reproduce the results?

Yes. And I splited the data based on the txt files in SSDA_MME/data/txt/multi you provided.
That is some kind of weird that in the experiments of one-shot setting, I can obtain similar results (i.e., the difference is smaller than 0.5%) in some adaptation scenarios (e.g., C to S, S to P), while in other cases (like, P to C, R to S), the differences are larger than 1.0%.
